resources/js/Components/Articles/ArticleCard/ArticleCard.tsx

```diff
@@ -1,34 +1,30 @@
+import { DateTime } from "@ardenthq/sdk-intl";
 import { useTranslation } from "react-i18next";
-import { FeaturedCollections } from "@/Components/Articles/ArticleCard/ArticleCard.blocks";
-import { type ArticleCollections } from "@/Components/Articles/ArticleCard/ArticleCardContracts";
+import { FeaturedCollections } from "@/Components/Articles/Article.blocks";
+import { Img } from "@/Components/Image";
 
-interface Properties {
-    // TODO(@alfonsobries)[2023-09-30]. Replace with a dynamic type once defined
-    article: {
-        title: string;
-    };
-    collections: ArticleCollections;
-}
-
-export const ArticleCard = ({ article, collections }: Properties): JSX.Element => {
+export const ArticleCard = ({ article }: { article: App.Data.Articles.ArticleData }): JSX.Element => {
     const { t } = useTranslation();
 
     return (
         <a
             data-testid="ArticleCard"
-            href="#"
-            className="group flex w-full max-w-[335px] flex-col overflow-hidden rounded-xl border border-theme-secondary-300 bg-white"
+            href={`/articles/${article.slug}`}
+            className="group flex w-full flex-col overflow-hidden rounded-xl border border-theme-secondary-300 bg-white"
         >
-            <div className="mx-2 mt-2 aspect-[16/9] overflow-hidden rounded-lg bg-theme-secondary-300">
-                <img
-                    className="w-full object-cover"
-                    alt="TBD"
-                    src="https://images.unsplash.com/photo-1638803040283-7a5ffd48dad5?ixlib=rb-4.0.3&ixid=M3wxMjA3fDB8MHxwaG90by1wYWdlfHx8fGVufDB8fHx8fA%3D%3D&auto=format&fit=crop&w=2592&q=80"
+            <div className="mx-2 mt-2 aspect-video overflow-hidden rounded-lg bg-theme-secondary-300">
+                <Img
+                    className="h-full w-full object-cover"
+                    wrapperClassName="h-full [&>span]:h-full"
+                    alt={article.title}
+                    src={article.image}
                 />
             </div>
 
             <div className="flex flex-1 flex-col px-6 py-3">
-                <div className="text-sm font-medium text-theme-secondary-700">24 Oct 2023</div>
+                <div className="text-sm font-medium text-theme-secondary-700">
+                    {DateTime.fromUnix(article.publishedAt).format("D MMM YYYY")}
```

Comment:
can you move this to a separate component/helper method so it's easy to reuse. It should also use the user's date setting and not hardcoded to this format

Reply:
Should be fine.